src/pudl/metadata/resources/ferc1.py

```diff
@@ -1719,6 +1765,7 @@
                 "record_id",
                 "is_disaggregated_utility_type",
                 "is_disaggregated_in_rate_base",
+                "table_name",
             ],
```

Comment:
nit: would be nice if this was a little more specific (something like `original_ferc1_table_name`) but not blocking!

Reply:
this is not a bad idea at all! but where this column name originally gets assigned is really upstream. so i'd like to ignore this for now if thats alright